tutorials/project_wizard/module_types.md

```diff
@@ -11,6 +11,8 @@ Create an empty plugin project,
 see 
 [Creating a Plugin Project](/basics/getting_started/creating_plugin_project.md).
 
+**Attention: "module" is IntelliJ-specific.**
```

Comment:
Once again this nees to be rephrased. Modules are not IntelliJ-specific; the UI for selecting a module type and creating a module through the project wizard is.

Reply:
Get